### DISCUSSION TITLE
Restart to version 1

### DISCUSSION THREAD

**Discussion by karlo-humanmanaged** [semantic-release/semantic-release#2706] [General]
Hi, I'm new to semantic release and was able to configure it to work with my pnpm monorepo after a lot of trial and error. The problem I have now is that the version of my packages are already quite high to be the first release. How do I reset the versions of the packages? I understand that this would involve deleting packages from our private npm instance (which is doable). We are also using a self-hosted Gitlab.

Do I just do the following:

1. Delete all generated CHANGELOG.md from our packages
2. Delete all tags created by semantic-release
3. Delete all packages pushed to our artifactory

Is there anything else I need to do?

My expectation is that the next commit, will have a version of v1.0.0

**Comment by glitchcowboy** [comment]
[ME TOO]
I'm deployed in gitlab.  For some reason when I delete all tags locally, and remotely and start fresh, semantic-release somehow picks up on a couple different old version series' that I had before and re-tags them.  I found some docs that reference git notes, but I can't find any notes in my repo that look relevant. 
If I start from scratch, I get a version 1.0.0
commit to main, I get version 1.0.1
If I delete all tags and make a new commit to main, I would expect 1.0.0, but it creates THREE tags: 1.0.0, 1.0.1, 1.0.2

Here's my config:
``` json
{
  "release": {
    "branches": [
      "+([0-9][0-9]?)?(.{+([0-9][0-9]?),x}).x",
      "main",
      "master",
      { "name": "fix/**", "prerelease": "${name.replace(/\\//g, '-')}" },
      { "name": "feat/**", "prerelease": "${name.replace(/\\//g, '-')}" }
    ],
    "plugins":  [
      "@semantic-release/commit-analyzer",
      "@semantic-release/release-notes-generator",
      "@semantic-release/gitlab"
    ],
    "tagFormat": "${version}"
  }
}
```

**Reply by glitchcowboy** [reply]
My problem was that my gitlab-runner was caching the repo and re-using it on subsequent runs (and not pruning tags).  The Gitlab CI/CD Git Strategy settings were set to Fetch with a Git shallow clone of 20.  Setting it to clone and a Git shallow clone of 0 solved the issue.  
Now when I delete tags they stay deleted and I'm able to restart my version numbering automatically at 1.0.0 or manually if I make a 0.0.0 tag
